src/core/parser_internal/func_worker.hpp

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include "docgen_worker.hpp"
+#include "ignore_worker.hpp"
+
+namespace docgen {
+namespace core {
+namespace parser_internal {
+
+struct FuncWorker_routines : private routine_details_t
+{
+	private:
```

Comment:
remove `private` and `protected`. Static classes don't even need to be inherited. See below.